### ISSUE TITLE
project active anymore?

### ISSUE THREAD

**Issue by miconx** [paidem/guacozy#52]
is this project active anymore ?
did not see any codechange for a while 

just curious if this would work with a current version of guacamole too (1.5.0)?

**Comment by paidem** [comment]
No, I do not develop it actively currently. Guess you’ll have to try if it works with newer version. On 2023-02-21, at 15:25, Norbert M Haigermoser ***@***.***> wrote:﻿
is this project active anymore ?
did not see any codechange for a while
just curious if this would work with a current version of guacamole too (1.5.0)?
